src/main/java/com/stripe/net/ApiResource.java

```diff
@@ -55,6 +55,14 @@ public ReflectionAccessFilter.FilterResult check(Class<?> rawClass) {
     return builder.create();
   }
 
+  public static String fullUrl(String defaultBaseUrl, RequestOptions options, String relativeUrl) {
```

Comment:
```suggestion
  protected static String fullUrl(String defaultBaseUrl, RequestOptions options, String relativeUrl) {
```

Reply:
Alas, StripeCollection is not a subclass of ApiResource